libqtile/window.py

```diff
@@ -151,13 +160,15 @@ def __init__(self, window, qtile):
         self._float_info = {
             'x': None,
             'y': None,
+            'z': NORMAL_LAYER,
```

Comment:
Why do you set this if you do not add a `float_z` property and play with it? I think we do not need a `float_z`, and thus do not need this addition. Unless we have a use case where a dedicated Z-value when the window is floating while keeping the original value aside makes sense.